pkg/cmd/deploy/bucket.go

```diff
@@ -64,6 +66,14 @@ func askForInput(msg string, defaultIn string) (string, error) {
 
 // replaceInvalidChars Regular expression to find disallowed characters: "[^a-zA-Z0-9]+" replace invalid characters with -
 func replaceInvalidChars(str string) string {
+	prefixes := []string{"azion-", "b2-", "azion-"}
```

Comment:
azion is repeated

Reply:
yes

Reply:
it is intentional not to check string but to prefix to simplify the code